Source/Scene/Expression.js

```diff
@@ -541,27 +541,39 @@ define([
 
     function parseKeywordsAndVariables(ast) {
         if (isVariable(ast.name)) {
-            return new Node(ExpressionNodeType.VARIABLE, getPropertyName(ast.name));
+            var name = getPropertyName(ast.name);
+            if (name.substr(0, 8) === 'tiles3d_') {
```

Comment:
I think it needs a sentence to explicitly say those names are reserved.  Help the implementer make unambitious decisions.